### PR TITLE
OSDOCS-10366 OCP Core 4.13.41 RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.13

Issue: [OSDOCS-10366](https://issues.redhat.com/browse/OSDOCS-10366)

Link to docs preview:

QE review: Not needed
- [ ] QE has approved this change.

Additional information: Expected merge date is May 2nd. Errata links will not work until then.

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -4073,3 +4073,36 @@ $ oc adm release info 4.13.40 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.13.41 --pullspecs
+----
+
+[id="ocp-4-13-41-bug-fixes"]
+==== Bug fixes
+
+* Previously, both tuned and irqbalanced were modifying the Interrupt Request (IRQ) CPU affinity which caused issues. With this release, only irqbalanced is in charge of configuring interrupt affinity. (link:https://issues.redhat.com/browse/OCPBUGS-31936[*OCPBUGS-31936*])
```
Comment (mburke5678) [inline release_notes/ocp-4-13-release-notes.adoc:4096]: Maybe anthromorphism??
```suggestion
* Previously, both tuned and irqbalanced were modifying the Interrupt Request (IRQ) CPU affinity which caused issues. With this release, only irqbalanced configures interrupt affinity. (link:https://issues.redhat.com/browse/OCPBUGS-31936[*OCPBUGS-31936*])
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -4073,3 +4073,36 @@ $ oc adm release info 4.13.40 --pullspecs
...
+
+* Previously, the autoscaler could panic and exit when MachineSets were defined with taints that did not contain the `value` field. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31702[*OCPBUGS-31702*])
```
Comment (mburke5678) [inline release_notes/ocp-4-13-release-notes.adoc:4098]:
```suggestion
* Previously, the autoscaler could panic and exit when machine sets were defined with taints that did not contain the `value` field. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31702[*OCPBUGS-31702*])
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -4073,3 +4073,36 @@ $ oc adm release info 4.13.40 --pullspecs
...
+
+* Previously, the console backend proxy server was sending operand list requests to the public API server endpoint. This caused Certificate Authority (CA) issues under some circumstances. With this release, the proxy configuration was updated to point to the internal API server endpoint which fixed this issue. (link:https://issues.redhat.com/browse/OCPBUGS-31595[*OCPBUGS-31595*])
```
Comment (mburke5678) [inline release_notes/ocp-4-13-release-notes.adoc:4100]:
```suggestion
* Previously, the console backend proxy server was sending operand list requests to the public API server endpoint. This caused Certificate Authority (CA) issues under some circumstances. With this release, the proxy configuration is updated to point to the internal API server endpoint, which fixed this issue. (link:https://issues.redhat.com/browse/OCPBUGS-31595[*OCPBUGS-31595*])
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -4073,3 +4073,36 @@ $ oc adm release info 4.13.40 --pullspecs
...
+
+* Previously, when the YAML editor was opened to edit a pipeline, the pipeline name would be overridden and changed to `new-pipeline`. With this release, the original pipeline name is preserved and not overridden when the YAML editor is opened. (link:https://issues.redhat.com/browse/OCPBUGS-31077[*OCPBUGS-31077*])
+
+* Previously, observability dashboards used expensive queries to show data which caused frequent timeouts on clusters with a large number of nodes. With this release, observability dashboards use recording rules that are precalculated to ensure reliability on clusters with a large number of nodes. (link:https://issues.redhat.com/browse/OCPBUGS-25922[*OCPBUGS-25922*])
```
Comment (mburke5678) [inline release_notes/ocp-4-13-release-notes.adoc:4104]:
```suggestion
* Previously, Observability dashboards used expensive queries to show data which caused frequent timeouts on clusters with a large number of nodes. With this release, observability dashboards use recording rules that are precalculated to ensure reliability on clusters with a large number of nodes. (link:https://issues.redhat.com/browse/OCPBUGS-25922[*OCPBUGS-25922*])
```

**Comment by mburke5678** [comment]
@cbippley A few comments. Otherwise LGTM
Please add a direct preview link if you need peer review.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 5]
```diff
@@ -4073,3 +4073,36 @@ $ oc adm release info 4.13.40 --pullspecs
...
+
+* Previously, the autoscaler could panic and exit when MachineSets were defined with taints that did not contain the `value` field. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31702[*OCPBUGS-31702*])
+
+* Previously, the console backend proxy server was sending operand list requests to the public API server endpoint. This caused Certificate Authority (CA) issues under some circumstances. With this release, the proxy configuration was updated to point to the internal API server endpoint which fixed this issue. (link:https://issues.redhat.com/browse/OCPBUGS-31595[*OCPBUGS-31595*])
+
+* Previously, when the YAML editor was opened to edit a pipeline, the pipeline name would be overridden and changed to `new-pipeline`. With this release, the original pipeline name is preserved and not overridden when the YAML editor is opened. (link:https://issues.redhat.com/browse/OCPBUGS-31077[*OCPBUGS-31077*])
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-13-release-notes.adoc:4102]: I think you're right.

**Pull request merged**